.github/workflows/load_test.yml

```
@@ -0,0 +1,38 @@
+on:
+  release:
+    types:
+      - prereleased
+  pull_request:
+    branches: [ feat/automatizeLoadTests, main ]
+
+name: Running Load Tests
+jobs:
+  load_tests:
+    name: unning Load Tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '^1.13.1'
+      - name: Installing dependencies yq and skaffold
+        run: |
+          sudo wget -O /usr/local/bin/yq https://github.com/mikefarah/yq/releases/download/3.3.0/yq_linux_amd64
+          sudo chmod +x /usr/local/bin/yq
+      - name: Setup Minikube
+        uses: manusa/actions-setup-minikube@v2.0.1
```

Could we bump this to the latest version?
```suggestion
        uses: manusa/actions-setup-minikube@v2.2.0
```

---

definetely